Week-1/catty-board/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <title>Catty Board</title>
+  </head>
+  <body>
+    <div id="root">
+      <nav class="nav">
+        <button><em class="la la-bars"></em></button>
+        <button><em class="la la-home"></em></button>
+        <button><em class="la la-search"></em></button>
+        <button><em class="la la-volume-up"></em></button>
+        <button><em class="la la-user"></em></button>
+        <button><em class="la la-cog"></em></button>
+        <button><em class="la la-spotify"></em></button>
+        <button><em class="la la-soundcloud"></em></button>
+      </nav>
+      <div class="root--primary">
+        <header class="header">
+          <section class="header--logo">
+            <picture>
+              <img src="./images/cattyboard.jpg" alt="Catty Board 2036 image" />
+            </picture>
+          </section>
+          <section class="header--primary">
+            <section class="header--primary-text">
+              <h2>CattyBoard Top 100 Single Charts(11.06.36)</h2>
+              <p>Unknown Artist</p>
+              <p>2016.Charts.100songs</p>
+            </section>
+            <section class="header--primary-buttons">
+              <button><em class="la la-play"></em> Play all</button>
+              <button><em class="la la-plus"></em> Add to</button>
+              <button><em class="la la-ellipsis-h"></em> More</button>
+            </section>
+          </section>
+        </header>
+        <section class="main">
+          <p id="caption" hidden>Table for cattyboard albums</p>
+          <table aria-describedby="caption" class="main--primary-table">
```

Comment:
in this case, using table was perfectly fine but I would suggest you to create similar kind of ui with flex or grid instead of tables. Using tables is not that common across the industry because of its cross browser and device compatibility issues.